process/templates/WG_NAME_incubating_stage.md

```diff
@@ -1,21 +1,43 @@
 ## Working Group incubation application
 
-### List of regular contributors
-The WG must have a minimum of 5 contributors from at least 3 different organizations attending regulary.
+### List WG Chair(s) and or Vice Chair
+The WG must have a minimum of 1 Chair
 
-  * "name, affiliation, GitHub ID"
+  * "name, affiliation, GitHUb ID"
```

Comment:
Is having a WG Chair actually a requirement?

It's not mentioned in https://github.com/ossf/tac/blob/main/process/TI-Gives%2BGets.md#givesrequirements.

It is referenced in https://github.com/ossf/tac/blob/main/process/working-group-lifecycle.md, but not explicitly required. If we want it to be required, we should probably add it to that doc under "To become `Incubating`".